Tasks/SqlAzureDacpacDeployment/Utility.ps1

```diff
@@ -9,56 +9,43 @@ function Check-ServerName
     }
 }
 
-function Get-AgentStartIPAddress
+function Get-AgentIPRange
 {
-    $endpoint = (Get-VstsEndpoint -Name SystemVssConnection -Require)
-    $vssCredential = [string]$endpoint.auth.parameters.AccessToken
-
-    $vssUri = $env:SYSTEM_TEAMFOUNDATIONCOLLECTIONURI
-    if ($vssUri.IndexOf("visualstudio.com", [System.StringComparison]::OrdinalIgnoreCase) -ne -1) {
-        # This hack finds the DTL uri for a hosted account. Note we can't support devfabric since the
-        # there subdomain is not used for DTL endpoint
-        $vssUri = $vssUri.Replace("visualstudio.com", "vsdtl.visualstudio.com")
-    }
+    param(
+        [String] $serverName,
+        [String] $sqlUserName,
+        [String] $sqlPassword
+    )
 
-    Write-Verbose "Querying VSTS uri '$vssUri' to get external ip address"
+    [hashtable] $IPRange = @{}
+    $sqlCmdArgs = "-S $serverName -U $sqlUsername -P $sqlPassword -Q `"select getdate()`""
 
-    # Getting start ip address from dtl service
-    Write-Verbose "Getting external ip address by making call to dtl service"
-    $vssUri = $vssUri + "/_apis/vslabs/ipaddress"
-    $username = ""
-    $password = $vssCredential
+    Write-Verbose "Executing sqlcmd.exe with arguments $sqlCmdArgs"
```

Comment:
This is done to get the IP address range. _select getdate()_ is a dummy script - user need not know about this.
better verbose log would be _Reaching to SQL Server to check IP Address exceptions using servername <> and username <>_ or something similar